### PR TITLE
Add sysconfig to Ranger executables

### PR DESCRIPTION
**Pull request by jack-altiscale**
- Need the values we set in sysconfig to be present in the Ranger executables
- Adjust logdir location

### CODE REVIEW

File: usr/bin/ranger-usersync [hunk 1]
```diff
@@ -25,7 +27,7 @@ action=`echo $action | tr '[:lower:]' '[:upper:]'`
 realScriptPath=`readlink -f $0`
 realScriptDir=`dirname $realScriptPath`
 cd $realScriptDir
-cdir=`pwd`
+cdir=/etc/ranger/usersync
```
Comment (cetin-altiscale) [inline usr/bin/ranger-usersync]: If possible, I prefer setting an env variable for /etc/ranger. Hard-coding might be a problem for the future releases.

File: usr/bin/ranger-usersync [hunk 2]
```diff
@@ -56,7 +58,7 @@ if [ "${action}" == "START" ]; then
   cd ${cdir}
   umask 0077
   if [ -z "${logdir}" ]; then 
-      logdir=${cdir}/logs
+      logdir=/var/log/ranger/usersync
```
Comment (cetin-altiscale) [inline usr/bin/ranger-usersync]: same here.

**Review by cetin-altiscale — APPROVED** [review]
I think this is good, but if possible, let's use configurable variables to refer to directories.

**Comment by jack-altiscale** [comment]
Great suggestion. I added the variables to sysconfig.

**Pull request merged**